### ISSUE TITLE
No scroll to navigate the diagram

### ISSUE THREAD

**Issue by lmcbout** [theia-ide/yangster#27]
No vertical/horizontal scroll to see the whole diagram.
![peek 2017-10-23 noscroll](https://user-images.githubusercontent.com/419051/31902241-271b03e0-b7f2-11e7-8406-49f8d6d6c7ea.gif)

**Comment by JanKoehnlein** [comment]
Sorry, but I consider scrollbars a thing of the past. They take a lot of space and are usually sluggish to use. Why do you miss them exactly?

**Comment by lmcbout** [comment]
I agree the scroll bar take some space, but we need a fast way to navigate and see a portion of the diagram. When the diagram has a lot of boxes in it, the space require to display the complete diagram will not fit into the screen display without scrolling

**Comment by hexa00** [comment]
Indeed and having them doesn't mean you have to use them.
